content/_guides/creatingchannels.md

```diff
@@ -1,52 +1,211 @@
 ---
 title: Creating Channels
 category: chanop
-credits: web7
 ---
 
+A person who joins an _empty_ and unregistered channel is granted
+**temporary** Operator status. This does not guarantee their ownership of the
+channel. They need to `REGISTER` the channel with `ChanServ` to gain ownership
+of it.
+
+This guide will help you prepare for and go through that process.
+
 ## Choosing the correct namespace
 
 Before creating a new channel on Libera.Chat, please take a moment to read up
 on [our namespaces policy](/chanreg) and ensure that your channel is named
 appropriately from the beginning. This will avoid disruptions in the future.
 
-## How to create a new channel
+## Be identified
+
+To register and manage channels, you will need to be [identified to](registration#logging-in)
+`NickServ`.
+
+## Pick a name
+
+Before doing that, you'll obviously need to pick a channel name! Think of one
+that is descriptive of the channel's purpose, and has no spaces or commas.
+The name you choose will become what's called your namespace. You can use
+some punctuation such as ".", "_" or "+" instead of spaces, but it is better
+to just drop the space.
+
+It is typical to reserve `-` as a delimiter for additional channels within the
+namespace, such as `-dev` or `-ops`. Using a `-` in the root name might result
+in conflicting namespaces, and cause complications for you and others when
+registering groups with Libera.Chat.
+
+Once you've settled on the name, add the appropriate number of `#`'s to the
+beginning, according to the rules in [our namespaces policy](/chanreg).
+
+For example, if it is an official representation of your project and the
+project is named "Awesome App", you can use one `#`. You'll probably go with
+something like: `#AwesomeApp`
+
+If it's not a "Project" or "Community" channel based on the definitions
+described in [our namespaces policy](/chanreg) you'd go with two `#`. As an
+example, it might be a channel for your friends to talk to you in, so you'd
+probably pick something like `##YourNickname`
+
+## Check that it is available
+
+You can check that a channel is registered or not with `ChanServ`. To see if
+for example `#libera` is registered, you would do:
 
-### Be identified
+```irc
+/msg ChanServ info #libera
```

Comment:
```suggestion
/msg ChanServ INFO #libera
```

I know they both work but we're pretty consistent across the site at capitalizing these commands

Reply:
Also if you join a channel, and it's empty, and you've got ops, then I think that means that it's available. This command could be useful for checking availability without joining, though.

Reply:
Yeah it's also easy to overlook it when typing. ty